auction-house/js/README.md

```diff
@@ -0,0 +1,18 @@
+# mpl-auction-house
+
+This package contains the auction house contract SDK code.
+
+## Developing
+
+In order to update the generated SDK when the rust contract was updated please run:
+
+```
+yarn gen:api
+```
+
+NOTE: at this point this only generates the IDL json file but will generate TypeScript
+definitions and SDK code as well, derived from that IDL.
+
+## LICENSE
+
+Apache v2.0
```

Comment:
The license file in this PR is MIT so this should match. The license files in the other contracts in this repo are also MIT. 
Note that the [License file in the root of the repo is Apache 2.0](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/LICENSE). Separately we'll want to iron this out.